.travis.yml

```diff
@@ -7,7 +7,7 @@ install:
     - pip install -r requirements-optional.txt coveralls
 script:
     - flake8 .
-    - py.test --cov=automation_tools tests/
+    - py.test --cov=automation_tools tests/test_log.py
```

Comment:
This reinforce my feeling of not having the upgrade tests on this repo. They are listed under the same directory and are not meant to test the automation_tools project itself